File: Assets/PACKT_Scripts/PlayerStatus_Final.js
#pragma strict

var health : int = 7;
var healthLimit : int = 10;

function AddHealth (increase : int)
{
	health += increase;
	
	if (health > healthLimit)
	{
		health = healthLimit; 
	}

}

function AddDamage (damage : int)
{	
	health-=damage;

	if (health <= 0)
	{
		health = 0;
		Die();
	}
}

function Die()
{
	Debug.Log("You Died");
}

File: Assets/PACKT_Scripts/PlayerStatus_Final.cs
using UnityEngine;
using System.Collections;

public class PlayerStatus_Final : MonoBehaviour {
	public int health = 7;
	public int healthLimit = 10;

	void AddHealth(int increase)
	{
		health += increase;

		if(health > healthLimit)
		{
			health = healthLimit;
		}
	}

	void AddDamage(int damage)
	{
		health -= damage;
		if(health <= 0)
		{
			health = 0;
			Die();
		}
	}

	void Die()
	{
		Debug.Log("You Died");
	}
}
